runtime/packages/core-utils/assembly/serialiser.ts

```diff
@@ -49,4 +49,12 @@ export class Serialiser {
         const buffer64 = ((valueSize as u64) << 32) | valuePtr;
         return buffer64;
     }
+
+    /**
+     * Serialize bytes
+     * @param value 
+     */
+    static serialiseBytes(value: u8[]): i32 {
```

Comment:
I think that it does not do `serialisation of bytes` but it rather returns the pointer of the bytes array. Maybe we can change the name of the function to something more appropriate